InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Logic.Modify.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using InternTrack.Core.Api.Models.Interns;
+using Moq;
+using Xunit;
+
+namespace InternTrack.Core.Api.Tests.Unit.Services.Foundations.Interns
+{
+    public partial class InternServiceTests
+    {
+        [Fact]
+        public async Task ShouldModifyInternAsync()
+        {
+            // given
+            DateTimeOffset randomDateTime = GetRandomDateTime();
+            Intern someIntern = CreateRandomIntern();
+            Intern inputIntern = someIntern;
+            Intern storageIntern = inputIntern.DeepClone();
+            inputIntern.UpdatedDate = randomDateTime.AddMinutes(1);
```

Comment:
Isn't this violating UpdatedDateIsNotRecent use-case? Because your datetimebroker returns randomDateTime and your updatedDate here is set to 1 minute more than the current date (the one that datetimebroker is returning)